src/components/atoms/Input/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { FC } from "react";
+
+interface IInput {
+  type: string;
+  name: string;
+  value?: string | number;
+  ref?: React.Ref<HTMLInputElement>;
+  placeholder?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  variant?: string;
+}
+
+const Input: FC<IInput> = ({
+  type,
+  name,
+  value,
+  ref,
+  placeholder,
+  onChange,
+  variant,
+}) => {
+  const searchInputStyle: string = "w-full h-full bg-transparent";
```

Comment:
here actually defining the type is not necessary and useless
https://betterprogramming.pub/when-to-declare-type-to-variables-in-typescript-d69a917ac1cf#:~:text=When%20to%20declare%20type%20to,we%20don't%20have%20to.